ts/store/reducers/notifications/installation.ts

```diff
@@ -37,6 +43,12 @@ const reducer = (
       return { ...state, token: action.payload };
     case getType(notificationsInstallationTokenRegistered):
       return { ...state, registeredToken: action.payload };
+    // clear the state when the authentication is not longer valid
+    case getType(sessionExpired):
+    case getType(sessionInvalid):
+    case getType(logoutRequest): // even if the logout fails
```

Comment:
Not sure about this case: say there is a network error, the app will eventually proceed to declare a logout even though the server never ack'd it?

Reply:
Yes it is
This is due to a limitation (bad design imho) of the reducer
see https://www.pivotaltracker.com/story/show/177923188